src/Components/NavBar/Menus/NewMobileNavMenu.tsx

```diff
@@ -0,0 +1,241 @@
+          <MobileLink href="/galleries">Galleries</MobileLink>
+          <MobileLink href="/fairs">Fairs</MobileLink>
+          <MobileLink href="/shows">Shows</MobileLink>
+          <MobileLink href="/institutions">Museums</MobileLink>
+          <MobileLink href="/gallery-partnerships">
+            Partner with Artsy
+          </MobileLink>
+          {user ? <LoggedLinks /> : <AuthenticateLinks />}
+        </ul>
+      </AnimatingMenuWrapper>
+    </MenuViewport>
+  )
+}
+
+const MenuViewport = styled.nav`
+  width: 100vw;
+  height: 100vh;
+  overflow: hidden;
+  position: relative;
+`
+
+export const AnimatingMenuWrapper = styled.div<{
+  isOpen: boolean
+}>`
+  background: white;
+
+  position: absolute;
+  width: 100vw;
+  height: 100vh;
+  overflow: hidden;
+  z-index: ${p => (p.isOpen ? 9999 : 0)};
+
+  top: 0;
+  left: 0; /* might be simpler to just animate this instead of the transform3d business */
+  padding: 1em;
+
+  transform: translate3d(${p => (p.isOpen ? "0" : "100%")}, 0, 0);
+  transition: transform 0.15s;
+`
+
+AnimatingMenuWrapper.displayName = "AnimatingMenuWrapper"
+
+const Menu = ({
+  isOpen,
+  title,
+  links,
+  path,
+  push,
+  pop,
+  showBacknav = true,
+}) => {
+  return (
+    <AnimatingMenuWrapper isOpen={isOpen}>
+      <Flex position="relative">
+        {showBacknav && <BackLink pop={pop} />}
+        <Sans size={["5", "6"]} color={color("black100")} mx="auto">
+          {title}
+        </Sans>
+      </Flex>
+      <ul>{links.map(link => renderLink({ link, path, push, pop }))}</ul>
+    </AnimatingMenuWrapper>
+  )
+}
+
+const BackLink = ({ pop }) => {
+  return (
+    <Box position="absolute" top="-6px">
+      <a
+        href="#"
+        onClick={e => {
+          e.preventDefault()
+          pop()
+        }}
+      >
+        <ChevronIcon
+          direction="left"
+          color={color("black100")}
+          height="10px"
+          width="10px"
+          top="7px"
+          left="5px"
+        />
+      </a>
+    </Box>
+  )
+}
+
+const renderLink = ({ link, path, push, pop }) => {
+  const isSubMenu = !!link.menu
+  if (isSubMenu) {
+    return (
+      <>
+        <MobileSubmenuLink
+          key={link.menu.title}
+          menu={link.menu}
+          path={path}
+          push={push}
+          pop={pop}
+        >
+          {link.text}
+        </MobileSubmenuLink>
+        {link.dividerBelow && <Separator my={1} color={color("black10")} />}
+      </>
+    )
+  } else {
+    return (
+      <>
+        <MobileLink key={link.href} href={link.href}>
+          {link.text}
+        </MobileLink>
+        {link.dividerBelow && <Separator my={1} color={color("black10")} />}
+      </>
+    )
+  }
+}
+
+export const MobileSubmenuLink = ({ children, menu, path, push, pop }) => {
+  console.log("MenuLink got", path)
+  return (
+    <li>
+      <Flex
+        py={0.5}
+        flexDirection="row"
+        onClick={() => {
+          push(menu.title)
```

Comment:
To reiterate an earlier conversation there is a potential glitch here in the (currently unlikely) case that to separate submenus have the same title. No action required now, but just a reminder...